server/migrate/Schema.java

```diff
@@ -293,15 +334,18 @@ public void print(IndentPrintWriter writer) {
             }
             if (when != null) {
                 writer.println(",");
-                writer.print("when ");
-                String ruleString = when.toString();
-                writer.print(ruleString.substring(0, ruleString.length() - 1));
+                when.print(writer);
             }
             if (then != null) {
-                writer.println(",");
-                writer.println("then ");
-                String ruleString = then.toString();
-                writer.print(ruleString.substring(0, ruleString.length() - 1));
+                writer.println(", then {");
+                writer.indent();
+                String then_string = then.toString();
+                writer.println(then_string.substring(2, then_string.length() - 3));
+                writer.unindent();
+//                writer.indent();
```

Comment:
btw are these meant to still be commented?

Reply:
No they are not; good catch